t/40-script_load_templates.t

```diff
@@ -62,7 +62,7 @@ $host     = "localhost:$mojoport";
 $pid      = OpenQA::Test::Utils::create_webapi($mojoport, sub { OpenQA::Test::Database->new->create; });
 $filename = 't/data/40-templates.json';
 $args     = "--host $host --apikey $apikey --apisecret $apisecret $filename";
-$expected = qr/Bad Request: No template specified/;
+$expected = qr/Bad Request: Invalid request parameters \(template\)/;
```

Comment:
The string "Invalid request parameters (template)", where is this defined? Somehow this reads like a template parameter was specified which isn't accepted but isn't the idea actually the opposite: That the template parameter is required but was *not* specified?

Reply:
`_validation_error` as defined in `lib/OpenQA/WebAPI/Plugin/Helpers.pm` renders the message if validation fails i.e. [Validation::failed](https://mojolicious.org/perldoc/Mojolicious/Validator/Validation#failed)

Reply:
I see. Maybe we should adjust the string in the helper to be more generic? Don't you consider it confusing as well that it says "invalid parameter" for a parameter tha was actually not specified?

Reply:
It's a step up from "wrong parameter" if you ask me. Suggestions welcome if you think there's nicer ways to convey this.

Reply:
It would generally be nice to distinguish between missing and invalid parameters.